packages/tsconfig.json

```diff
@@ -1,17 +0,0 @@
-{
```

Comment:
This file is (was?! i didn't have the chance to test it) necessary in various IDEs for autocomplete and so on. Monorepos confuse IDEs and text editors :-/

Reply:
vs code seems fine without, didn't notice any effect. I'm not sure why would an ide read it either, it's not topomost to any ts code :thinking:  can you test? Or would you rather I just add it back?

Reply:
restored